tests/validator20/validate_spec_test.py

```diff
@@ -423,3 +425,13 @@ def test_highlight_inconsistent_schema_object_validation(minimal_swagger_dict, s
     minimal_swagger_dict.update(swagger_dict_override)
     with pytest.raises(SwaggerValidationError):
         validate_spec(minimal_swagger_dict)
+
+
+def test_validation_performance(minimal_swagger_dict):
```

Comment:
Please update test name to reflect what you're testing (so that you're actually caching the result of `read_file`)